rules/windows/process_creation/proc_creation_win_browsers_inline_file_download.yml

```
@@ -0,0 +1,41 @@
+title: File Download From Browser Process Via Inline Link
+id: 94771a71-ba41-4b6e-a757-b531372eaab6
+status: test
+description: Detects execution of a browser process with a URL argument pointing to a file with a potentially interesting exntesion. Can be abused to download arbitrary file and to trick the user.
```

```suggestion
description: Detects execution of a browser process with a URL argument pointing to a file with a potentially interesting extension. This can be abused to download arbitrary files or to trick the user into ??.
```
Couldn't figure out what the trick is. Others may wonder as well :)

(You can just edit this post to add the answer)

---

The "trick" isn't in this command. Basically the referenced tweet used a minimized window to hide from the user when downloading stuff (that's one way of doing it) but its part of a larger command, that's not necessarily captured via process creation logs.

I'll change it to "hide from the user" and give the minimized example.